File: Examples/ManualExample/ManualExample/ViewController.m
//
//  ViewController.m
//  ManualExample
//
//  Created by Tony Xiao on 6/30/16.
//  Copyright © 2016 Segment. All rights reserved.
//

#import "Analytics.h"
#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [SEGAnalytics track:@"Manual Example Main View Loaded"];
    [SEGAnalytics flush];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)fireEvent:(id)sender {
    [SEGAnalytics track:@"Manual Example Fire Event"];
    [SEGAnalytics flush];
}

@end

File: Examples/CocoapodsExample/CocoapodsExample/ViewController.swift
//
//  ViewController.swift
//  CocoapodsExample
//
//  Created by Tony Xiao on 6/30/16.
//  Copyright © 2016 Segment. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        Analytics.track("Cocoapods Example Main View Load")
        Analytics.flush()
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func fireEvent(sender: AnyObject) {
        Analytics.track("Cocoapods Example Fire Event")
        Analytics.flush()
    }

}

